### DISCUSSION TITLE
OIDCOAuthAcceptTokenAs: Unable to set cookie with custom name in 2.4.16.3

### DISCUSSION THREAD

**Discussion by bbartke** [OpenIDC/mod_auth_openidc#1261] [Q&A] [answered]
Hi,
the configuration directive to set a custom cookie name for OIDCOAuthAcceptTokenAs is not working in 2.4.16.3. It was properly working with 2.4.15.7:
```
# Define the ways in which bearer OAuth 2.0 access tokens can be passed to this Resource Server
OIDCOAuthAcceptTokenAs header cookie:access_token query
```
Error:
```
AH00526: Syntax error on line ... of ...:
Invalid value for directive 'OIDCOAuthAcceptTokenAs': invalid value 'cookie:access_token', must be one of ['header'|'post'|'query'|'cookie'|'basic']
```
Following the doc, the syntax did not change and should be correct:
https://github.com/OpenIDC/mod_auth_openidc/blob/master/auth_openidc.conf#L506-L514
Could this be a bug?

**Comment by zandbelt** [comment, marked as answer]
yes, that is a bug indeed, fixed in f7beb71a9a867ebda6cfcf69433c8cd96dd4bf53, thanks for reporting; however since this functionality is deprecated in favor of [mod_oauth2](https://github.com/OpenIDC/mod_oauth2), we won't prioritize it for a new release (unless our commercial constituency pushes for it)

**Comment by zandbelt** [comment]
this is now released in https://github.com/OpenIDC/mod_auth_openidc/releases/tag/v2.4.16.4
